README.md

```diff
@@ -1,18 +1,14 @@
 # Java Style Guide
-This project contains all style guides used for Java language files.
+In Sahab, we follow a code style that is mostly based on [Google code style](https://google.github.io/styleguide/javaguide.html) with a few customizations.  
+This Github project contains some config files related to that style:
 
-## Code Style
-Our code style is based on [Google code style](https://google.github.io/styleguide/javaguide.html) with a
-few customizations. The exact customizations and the reason behind each one of them is described below.
-To check code style on source codes we use [Checkstyle](https://checkstyle.sourceforge.io/). Currently, version 8.45.1
+-  checkstyle.xml: The config file for [Checkstyle](https://checkstyle.sourceforge.io/). Using this file and its associated tools like Maven and Gradle plugins, we can ensure our code style rules in our CI/CD pipelines. To make this file, we have used [the google check style file](https://github.com/checkstyle/checkstyle/blob/master/src/main/resources/google_checks.xml) as the base file and we have changed it a little according to our customizations. Currently, version 8.45.1
 of Checkstyle is recommended.
+-  intellij-codestyle.xml: The config file to configure our code style in Intellij. To make this file, we have used [the google Intellij code style file](https://github.com/google/styleguide/blob/gh-pages/intellij-java-google-style.xml) as the base file and we have changed it a little according to our customizations.
```

Comment:
Please add the code style file of VSCode too. (I think you can get it from Amin or other developers and it is straightforward to do it in this PR)